worlds/stardew_valley/test/bases.py

```diff
@@ -179,6 +180,16 @@ def assert_cannot_reach_location(self, location: Location | str, state: Collecti
             state = self.multiworld.state
         super().assert_cannot_reach_location(location, state)
 
+    def assert_can_reach_region(self, region: Region | str, state: CollectionState | None = None) -> None:
+        if state is None:
+            state = self.multiworld.state
+        super().assert_can_reach_region(region, state)
+
+    def assert_cannot_reach_region(self, region: Region | str, state: CollectionState | None = None) -> None:
+        if state is None:
+            state = self.multiworld.state
+        super().assert_cannot_reach_region(region, state)
```

Comment:
```suggestion
        return super().assert_can_reach_region(region, state)

    def assert_cannot_reach_region(self, region: Region | str, state: CollectionState | None = None) -> None:
        if state is None:
            state = self.multiworld.state
        return super().assert_cannot_reach_region(region, state)
```
super returns should be forwarded unless they're meant to be modified. (I know it's just a None here, but this safes one from weird errors after later refactors when subclass methods don't update to the new return type)

Reply:
Yeah good catch, I'll change it when I'm home